src/components/Datepicker/Datepicker.tsx

```diff
@@ -97,36 +99,39 @@ export const Datepicker: FC<DatepickerProps> = ({
   labelClearButton = 'Clear',
   showTodayButton = true,
   labelTodayButton = 'Today',
-  defaultDate = new Date(),
+  defaultValue = new Date(),
```

Comment:
The default value should be empty so `placeholder` is displayed with just `<Datepicker placeholder=".." />`.

You are just keeping it the same as it was before, so I'm not being critical of anything here - it would just be helpful if you could make that change while we're at it :D

Reply:
Dun, considering "placeholder" is actually "label".